eng/nuget/Microsoft.NET.Runtime.Emscripten.Cache/Microsoft.NET.Runtime.Emscripten.Cache.props

```diff
@@ -0,0 +1,11 @@
+<Project>
```

Comment:
Does this need to be a separate file?

Reply:
Not sure.  I'm copying what the Emscripten.Sdk.pkgproj did